<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Клиент</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1011</width>
      <height>581</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="bScan">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>130</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать сканирование</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bReport">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>190</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Отчет</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pScanMonit">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>220</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>яНеЗнаюСлова</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bStop">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>160</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Остановить сканирование</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbDir">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>100</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбор папки</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbFile">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>70</y>
        <width>201</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Выбор файла</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="bChose">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>20</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>921</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>771</width>
        <height>441</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>520</y>
        <width>771</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QPushButton" name="bStartServ">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>530</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Запустить сервис</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bStopServ">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>500</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Остановить сервис</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="bToQuar">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>20</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Переместить в карантин</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bReturnQuar">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>50</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Вернуть из карантина</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bDelete">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>80</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bBack">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>520</y>
        <width>201</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Возврат</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lQuar">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>371</width>
        <height>521</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>0</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Обнаруженные угрозы</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Карантин</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lDetec">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>391</width>
        <height>521</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="bBackSM">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>530</y>
        <width>191</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Возврат</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>921</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="bChose_2">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>10</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QListWidget" name="lMonit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>391</width>
        <height>471</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="lTime">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>80</y>
        <width>391</width>
        <height>471</height>
       </rect>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="bMonit">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>80</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Начать отслеживание</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bMonitEnd">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>110</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Закончить отслеживание</string>
      </property>
     </widget>
     <widget class="QPushButton" name="bTime">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>260</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Запланировать проверку</string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>230</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="maximumTime">
       <time>
        <hour>23</hour>
        <minute>59</minute>
        <second>59</second>
       </time>
      </property>
      <property name="displayFormat">
       <string>HH:mm</string>
      </property>
      <property name="timeSpec">
       <enum>Qt::LocalTime</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>290</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Отменить проверку</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Мониторинг</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>351</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Запланированные проверки</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1011</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
